core/src/torii/mod.rs

```diff
@@ -364,6 +395,24 @@ async fn handle_subscription(events: EventsSender, stream: WebSocket) -> eyre::R
     Ok(())
 }
 
+/// Response body for get metrics request
+#[derive(Clone, Copy, Debug, Deserialize, Serialize)]
+pub struct Metrics {
+    /// Number of connected peers
+    peers: u64,
+    /// Number of committed blocks
+    blocks: u64,
+}
+
+#[allow(clippy::unused_async)]
```

Comment:
Since this function has no `await` can it be made a regular function, rather than `async`?

Reply:
Probably warp needs this signature, but, yes, we should check if it is possible to have it in other way.